aten/src/ATen/native/cuda/ScatterGatherKernel.cu

```diff
@@ -109,7 +109,6 @@ struct _cuda_scatter_gather_internal_kernel {
     auto offset_calc = make_offset_calculator<3>(iter);
     auto loop = [=]C10_DEVICE(int i) {
       auto offsets = offset_calc.get(i);
-
```

Comment:
This PR doesn't need to touch this file.
Please use `git checkout master -- aten/src/ATen/native/cuda/ScatterGatherKernel.cu` to undo the changes to the file.